frontend/src/api/GetBooksFromApi.tsx

```diff
@@ -63,8 +63,8 @@ function GetBooksFromApi(props: GetBooksFromApiProps) {
                        autoComplete={"off"}
                        onChange={handleChange}
                 />
-                <button className="search-button" type={"submit"}>Search</button>
-                <button className="search-button" onClick={handleClick}>Search by: {searchBy}</button>
+                <button className="button button-search" type={"submit"}>Search</button>
+                <button className="button button-search-by" onClick={handleClick}>Search by: {searchBy}</button>
```

Comment:
The class "button" is not needed, in your css file you can address the element with "button .button-search"

Reply:
Danke für deine Bemerkung, das war aber bewusste Entscheidung nicht über button Objekt zu definieren sondern über die Klasse, falls ganz andere Button style dazu kommen soll. Ich lasse dies erstmal so, sonst muss ich wieder viele Komponente anfassen.